### DISCUSSION TITLE
Lost debug adapter cable

### DISCUSSION THREAD

**Discussion by bhabas** [bitcraze/discussions#807] [Q&A]
Hello,

I am currently using the Crazyflie with the debug adapter kit for a project. Unfortunately, I've lost the [P3/Crazyflie 2.0 SWV cable](https://www.bitcraze.io/documentation/hardware/debug_adapter/debug_adapter-datasheet.pdf) that connects the Crazyflie to the debug adapter though.

Is there a standard name for the connection ends of this cable such that I could order one locally or would I need to purchase a full new debug adapter?

**Comment by ataffanel** [comment]
On the Crazyflie side the connector is `SM06B-SRSS-TB`, so it is a 6 pin SH connector from JST. Looking for the connector at Digikey, there seems to be some pre-assembled jumper-cable available for this series of connector.

**Reply by bhabas** [reply]
That's perfect! Thank you!
